package.json

```diff
@@ -16,9 +16,9 @@
   "dependencies": {
     "@buf/cosmos_ibc.bufbuild_es": "1.7.2-20240123184419-30c7be3837b0.1",
     "@buf/cosmos_ibc.connectrpc_es": "1.3.0-20240123184419-30c7be3837b0.1",
-    "@buf/penumbra-zone_penumbra.bufbuild_es": "1.7.2-20240201164939-04d44cf9ba58.1",
-    "@buf/penumbra-zone_penumbra.connectrpc_es": "1.3.0-20240201164939-04d44cf9ba58.1",
-    "@bufbuild/protobuf": "^1.7.1",
+    "@buf/penumbra-zone_penumbra.bufbuild_es": "1.7.2-20240202021444-2081ef4dca12.1",
+    "@buf/penumbra-zone_penumbra.connectrpc_es": "1.3.0-20240202021444-2081ef4dca12.1",
```

Comment:
So that we get the latest version of `TransactionPlannerRequest`, which includes the `feeMode` property.